etc/a.gwf

```diff
@@ -213,6 +232,19 @@ propose_add_family=yes
 # proposed in the welcome page.
 propose_place_surname=yes
 
+# provide a list of names for the decomposition of places (comma separated)
+# other = ""; town = ""; township = ""; canton = ""; district = "";
+# county = ""; region = ""; country = ""
+#
+#place=autre, ville, agglomération, canton, district, conté, région, pays
+
+# Spécific à to the API (?? TODO document ??)
+#places_format=
+
+# Allow personnal template files (yes/no)
+# The file name will be provided as an evar argument in the url ;ptempl=filename;
+#ptempl=no
```

Comment:
Do you have a use case?

I would not add it to `a.gwf` and silently remove it from geneweb's code.